zebrad/src/components/sync/tests/timing.rs

```diff
@@ -11,6 +11,7 @@ use tokio::time::{timeout, Duration};
 
 use zebra_chain::parameters::{Network, POST_BLOSSOM_POW_TARGET_SPACING};
 use zebra_network::constants::{DEFAULT_CRAWL_NEW_PEER_INTERVAL, HANDSHAKE_TIMEOUT};
+use zn::constants::INVENTORY_ROTATION_INTERVAL;
```

Comment:
Minor: I think this line could be merged with the line above it.

Reply:
This turns out to be slightly more complicated and very confusing, due to a wildcard import and module alias.

Fixed in a1f8d50af.